src/videoCache.js

```diff
@@ -42,17 +43,22 @@ const ttlBufferInSeconds = 15;
  * @param {string} impUrl An impression tracker URL for the delivery of the video ad
  * @return A VAST URL which loads XML from the given URI.
  */
-function wrapURI(uri, impUrl) {
+function wrapURI(uri, impUrl, arrayVastTrackers) {
```

Comment:
no need to include the type in the name
```suggestion
function wrapURI(uri, impUrl, vastTrackers) {
```

Reply:
done